### PR TITLE
Sequel benchmarks for rails

### PR DESCRIPTION
**Pull request by shahsaurabh0605**
I have not included the sequel equivalent of `bm_activerecord_scope_all_with_default_scope.rb` because there is no such concept of `default_scope` in sequel.
Also, sequel doesn't provide the `exclusion` helper, so I removed it from the validations.

### CODE REVIEW

File: rails/benchmarks/bm_sequel_finders_find_by_attributes.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+require 'bundler/setup'
+require 'sequel'
+require_relative 'support/benchmark_rails'
+
+DB = Sequel.connect(ENV.fetch('DATABASE_URL'))
+
+DB.create_table(:users) do
+    primary_key :id
```
Comment (tgxworld) [inline rails/benchmarks/bm_sequel_finders_find_by_attributes.rb]: Nitpick: Two spaces for indentation :)

File: rails/benchmarks/bm_sequel_finders_find_by_attributes.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+}
+
+1000.times do
+  User.create(attributes)
+end
+
+User.create(name: 'kir', email: 'shatrov@me.com')
+
+Benchmark.rails("sequel/#{db_adapter}_finders_find_by_attributes", time: 5) do
+  User.find(:email=>'shatrov@me.com')
```
Comment (tgxworld) [inline rails/benchmarks/bm_sequel_finders_find_by_attributes.rb]: Nitpick: prefer `email: 'shatrov@me.com'`

**Comment by shahsaurabh0605** [comment]
Done :+1:

**Comment by tgxworld** [comment]
@shahsaurabh0605 Good job :+1: . Before I review. Can you make sure we can run all sequel benchmarks using `driver.rb`?

**Comment by shahsaurabh0605** [comment]
All the sequel benchmarks are running provided I keep `null: true` for timestamp columns. Should I make that change?

**Comment by tgxworld** [comment]
@shahsaurabh0605 Ok I missed this. We have to move sequel away from the `rails` folder. Anything in the rails folder means it depends on Rails. Can you move the sequel benchmarks into it's own folder like bundler? We'll benchmark sequel across releases and on a per commit basis.

**Comment by shahsaurabh0605** [comment]
I am not sure on the dependencies as in what the other directory will contain. Anyways I will try and submit a PR.

**Comment by tgxworld** [comment]
Each folder correspond to a particular repository. So having sequel in the Rails folder is a little confusing because we're benchmarking Sequel against a moving Rails repository. What version should Sequel be fixed to? Instead, we'll just benchmark Sequel and have the ability to compare different benchmarks across repository on RubyBench.org

**Pull request closed without merging**